addons/website/static/src/scss/website.ui.scss

```diff
@@ -232,6 +232,9 @@ body .modal {
                 border-radius: 0;
             }
         }
+        .mjs-nestedSortable-error {
```

Comment:
Correct me if I am wrong (as it does not seems to be the case everywhere on the file) but isn't a blank line needed here?